packages/kit/src/loader/nuxt.ts

```diff
@@ -54,6 +54,19 @@ export async function loadNuxt (opts: LoadNuxtOptions): Promise<Nuxt> {
     envConfig: opts.dotenv // TODO: Backward format conversion
   })
 
+  // Mock new hookable methods
+  nuxt.removeHook = nuxt.clearHook.bind(nuxt)
+  nuxt.removeAllHooks = nuxt.clearHooks.bind(nuxt)
+  nuxt.hookOnce = (name: string, fn: (...args: any[]) => any) => {
```

Comment:
For sake of forward compatibility, (if hookable will support 3rd options arg) we could pass it here.

Also maybe would be better to conditionally override each to make sure we don't dobule/override polyfills?